main.nf

```diff
@@ -113,24 +114,29 @@ process buildHeader {
     # Add the two headers together and add the column names
     cat temp_header.txt contigs.txt > final_header.txt
     tail -n 1 vcf_header.txt >> final_header.txt
+    cat final_header.txt variants_remapped_sorted.vcf > variants_remapped_sorted_with_header.vcf
     """
 }
 
 /*
- * Add header to output VCF and merge with reference allele file
+ * Add header to unmapped variant VCF records
  */
-process mergeHeaderAndContent {
+process mergeOriginalHeaderAndVCFAndOutput {
```

Comment:
I think having the details in the comment description and a more general name helps to read the pipeline easier. Like renaming the process to something like `generateUnmappedVCF` and then explaining in the description that we are doing the merging

Reply:
I've address this comment in subsequent PR [EVA-2451 - allow novel ref allele #29
](https://github.com/EBIvariation/variant-remapping/pull/29)